Sources/Testing/testMacro.js

```diff
@@ -451,6 +459,20 @@ test('Macro methods object tests', (t) => {
     'Getter shall return a copy'
   );
 
+  myTestClass.setMyParameterizedObjectProp({ foo: 12, bar: 89 });
+  t.deepEqual(
+    myTestClass.getMyParameterizedObjectProp(),
+    { foo: 12, bar: 89 },
+    'Setter changes on a different object'
+  );
+
+  myTestClass.setMyParameterizedObjectProp(16, 52);
```

Comment:
`(16)` is ambiguous. If there is only 1 argument then I treat that as setting the property directly. Deciding what to do based on the argument type is not a good idea, because what if the property's value is itself an object?

I'll add a test for 3 arguments.